modules/auxiliary/scanner/http/dell_idrac.rb

```diff
@@ -81,7 +92,43 @@ def do_login(user = nil, pass = nil)
       )
       return :next_user
     else
-      print_error("#{target_url} - Dell iDRAC - Failed to login as '#{user}' with password '#{pass}'")
+      vprint_error("#{target_url} - Dell iDRAC - Failed to login as '#{user}' with password '#{pass}'")
+    end
+  end
+
+  def do_login_v9(user = nil, pass = nil)
+    uri = v9_url
+    auth = send_request_cgi({
+      'method' => 'POST',
+      'uri' => uri,
+      'SSL' => true,
+      'headers' => { 'user' => user, 'password' => pass },
+      'vars_post' => {
+        'user' => user,
+        'password' => pass
+      }
+    })
+    unless auth
+      print_error('iDRAC failed to respond to login attempt')
+      return :next_user # assume this is a temporary error
+    end
+    json = JSON.parse(auth.body)
+    if json['authResult'] == 1
+      vprint_error("#{target_url} - Dell iDRAC - Failed to login as '#{user}' with password '#{pass}'")
+      unless json['blockingTime'] == 0
+        vprint_error("\tServer throttled logins at #{json['blockingTime']} seconds")
+      end
```

Comment:
Do we need to react to this in some way, i.e. trying different users / sleeping / etc?